src/components/MyAccount/index.tsx

```diff
@@ -112,6 +116,12 @@ const MyAccountContent = ({
     }
   }, [organizations, loadedStringsForLocale]);
 
+  useEffect(() => {
+    if (userPreferences?.preferredWeightSystem) {
```

Comment:
might be easier to use https://github.com/terraware/terraware-web/blob/main/src/api/preferences/preferences.ts#L71
i still don't know what's better yet.. 
i have been using ^^
same for update

Reply:
ignore this ^^ i will revisit the pattern later